<launch>

	<!-- <node pkg="topic_tools" type="mux" name="mux_pointcloud" args="/camera/depth/color/sel_pointcloud /camera/depth/color/points none mux:=mux_pointcloud"/>
	<node pkg="topic_tools" type="mux" name="mux_image_rect" args="/camera/color/sel_image_rect /camera/color/image_rect none mux:=mux_image_rect"/> -->
	
	<include file="$(find homestri_moveit_config)/launch/demo.launch" >
	</include>

	<!-- <node pkg="shr_manipulation" type="manipulation_action_server.py" name="manipulation_action_server" output="screen"/> -->
	<!-- <node pkg="shr_behavior_trees" type="behavior_tree_server" name="behavior_tree_server" output="screen"/> -->


</launch>